js/templates/item.html

```diff
@@ -41,6 +41,11 @@ <h2 class="page-contractTitle noOverflow textOpacity1 marginLeft20 marginBottom8
             <%= polyglot.t('BUYNOW') %>
           </div>
         </li>
+        <% if (ob.totalReviews) { %>
+        <li class="border0 alignCenter padding10 marginBottom12 js-itemRating">
+          <span class="marginRight3"><%= ob.totalReviews %> Review<% ob.totalReviews !== 1 && print('s') %></span><% print(ob.starsTemplate({ rating: ob.avgRating })) %><span class="marginLeft2">(<%= ob.avgRating %>/5)</span>
```

Comment:
Using the print s is smart, but it won't work for translations. It should just always use the polyglot phrase for Reviews, even if there is only one.